### PR TITLE
CP-41461: add vm_setup_scripts in ACK repo

### PR DESCRIPTION
**Pull request by alexhimmel**
Signed-off-by: alexhimmel <alex.ac@qq.com>

### CODE REVIEW

**Comment by alexhimmel** [comment]
@DeliZhangX @xihuan-citrix please help to review, has tested 
Has created a VM with Rocky8.6 Linux and setup this VM with using the script "init-run.sh", and upload to https://repo.citrite.net/artifactory/incoming/tianxia/12-13/vpx-dlvm.tar.gz
Run tests with using above VM, Job Id: 3653534 ~ 3653538, All passed

File: vm_setup_scripts/init-run.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# This script is used to setup Driod VM template base on Rocky 8 Linux 8.6
+
+# install dependencies for test tools
+dnf update -y
+dnf install perl -y
+dnf --enablerepo=powertools install perl-List-MoreUtils -y
+dnf --enablerepo=powertools install perl-Readonly -y
+
```
Comment (DeliZhangX) [inline vm_setup_scripts/init-run.sh:9]: Where is test tool?

Comment (alexhimmel) [reply vm_setup_scripts/init-run.sh:9]: Test tool will be installed during test as before.

Comment (DeliZhangX) [reply vm_setup_scripts/init-run.sh:9]: you need to update doc https://info.citrite.net/pages/viewpage.action?pageId=1604229299 to ensure consistent

Comment (alexhimmel) [reply vm_setup_scripts/init-run.sh:9]: Sure, has updated the doc, Thanks.

Comment (DeliZhangX) [reply vm_setup_scripts/init-run.sh:9]: `, and user needs to install them to VM manually` should be removed.

**Review by DeliZhangX — APPROVED** [review]

File: vm_setup_scripts/init-run.sh (hunk 1 above)
Comment (alexhimmel) [reply vm_setup_scripts/init-run.sh:9]: Has removed

**Review by debugman42 — APPROVED** [review]

**Pull request merged**